api/src/messages/controllers/messages.controller.ts

```diff
@@ -38,7 +39,7 @@ export class MessagesController {
     @Post()
     async create(
         @Body() createMessageDto: CreateMessageDto,
-        @UserReq("id") userId: number
+        @UserReq("id", ParseIntPipe) userId: number
```

Comment:
Remove ParseIntPipe and fix UserReq decorator.

Reply:
i think it's good practise, cause if on frontend the string will be assign to JWT, here i can transform it to 'number' type

Reply:
What? Look/debug what is under the hood. 
This is implemented by You and there always will be number. As long as I know variable request.user is defined when You do `generateJwt(user: User)`. 
And there is also HUGE security vulnerability because You pass `user` to this function with plain texted password. So everyone who get user JWT token can easly get user's password. And I remember I paid attention to it in one of my first CR. And You make this mistake again :(

Reply:
Ye, that's right, i can remove this pipe.

But for second issue u don't have right, because i don't pass password to JWT
```ts
const { password, ...rest } = user;

return this.generateJwt(rest);
```

Reply:
```
    async login(userCredentialsDto: UserCredentialsDto) {
        const { id } = await this.userService.findOneByUsername(
            userCredentialsDto.username
        );

        return this.generateJwt({ ...userCredentialsDto, id });
    }
```

Reply:
```ts
@Column({ select: false })
    password?: string;
```

Password must be select by special query in that case, isn't select

Reply:
I remove this Pipe in all controllers

Reply:
But to generate this JWT from payload not from db query result!?

Reply:
it's fixed.